framework/api_vulkan_sample.cpp

```diff
@@ -454,6 +454,14 @@ VkPipelineShaderStageCreateInfo ApiVulkanSample::load_shader(const std::string &
 		shader_extension = ".spv";
 	}
 
+	if (get_shading_language() == vkb::ShadingLanguage::SLANG)
+	{
+		shader_folder = "slang";
+		// slang shaders are offline compiled to SPIR-V, so source is SPV
+		src_language     = vkb::ShaderSourceLanguage::SPV;
+		shader_extension = ".spv";
+	}
+
```

Comment:
As part of those two cases are identical, maybe it's better to have something like
```
auto shading_language = get_shading_language();
if ((shading_language == vkb::ShadingLanguage::HSLS) || (shading_language == vkb::ShadingLanguage::SLANG))
{
    shader_folder = (shading_language == HLSL) ? "hlsl" : "slang";
    ...
}
```

Reply:
I'd prefer to have them separate for better readability. I'll probably rework that into a switch/case though.

Reply:
Regarding the better readability: I find it better readable, if there are no identical code sequences for the different cases. You (as a human) have to carefully scan the two code sequences, just to realize they actually are identical. In such a case, I would always ask if that's correct